app/views/good_job/shared/_navbar.erb

```diff
@@ -60,7 +60,7 @@
 
         <li class="nav-item d-flex flex-column justify-content-center">
           <div class="form-check form-switch m-0">
-            <%= check_box_tag "live_poll", params.fetch("poll", 30), params[:poll].present?, role: "switch", class: "form-check-input" %>
+            <%= check_box_tag "live_poll", params.fetch("poll", 30), params[:poll].present?, role: "switch", class: "form-check-input", disabled: !GoodJob.configuration.dashboard_live_poll_enabled %>
```

Comment:
I think we also want to make sure that it is always unchecked when disabled to:

```suggestion
            <%= check_box_tag "live_poll", params.fetch("poll", 30), (GoodJob.configuration.dashboard_live_poll_enabled && params[:poll].present?), role: "switch", class: "form-check-input", disabled: !GoodJob.configuration.dashboard_live_poll_enabled %>
```

Reply:
nice, this looks good